lib/client.js

```diff
@@ -19,6 +19,23 @@ function validateKey(key, operation) {
     misc.assert(key.length < 250, 'Key must be less than 250 characters long');
 }
 
+function validateNamespaceKey(key, operation) {
+    validateKey(key, operation);
+    misc.assert(key.length < 230, 'Key must be less than 230 characters long when using namespaces');
+}
+
+function keyWithPrefix(key, prefix) {
+    if (_.isUndefined(prefix)) {
+        // only prefix was supplied
+        // return a curried funtion that expects the key
+        prefix = key;
+        return function(key) {
+            return _.toString(prefix) + '_' + key;
```

Comment:
```suggestion
            return _.toString(prefix) + ':' + key;
```

Reply:
Let's please keep with the colon as the separator for namespaces as suggested by the [Memcached article you linked](https://github.com/memcached/memcached/wiki/ProgrammingTricks#namespacing)

That's also the way I've always seen it done in production